CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 8.2
 -----
-
+*   Updates
+    *   Upgraded Compose and the Wear OS libraries to the latest versions 
```

Comment:
❓ Should we start announcing library updates in our changelog? My understanding is that it’s meant primarily for user-facing changes. I realize the Compose update is significant, but I don’t think it matters to our users as a library update.